docs/replaced_status.md

```diff
@@ -0,0 +1,26 @@
+# Replaced Status
+Accessioned objects have a special status of replaced that allows them to share identifying properties like md5sum or accession.  This essentially allows us to merge two objects. Here is a [link](https://docs.google.com/presentation/d/1uMDHw0MQBb3mMxOWm88cSq6aRWbn2KpZyHHI5yQMQAE/edit?pli=1&slide=id.g7171591498_0_295#slide=id.g7171591498_0_295) to ENCODE status slides in general.
+
+## Why do we have a replaced status
+Each schema has a list of identifying properties. These properties can be used in the url to access the object (see examples).  The values in these properties must be unique so that each value resolves to one and only one object.  However, there are many situations where you would want to have more than one object "merge" or all be directed to the same object.  To achieve that, we use the replaced status to release all identifying properties except the uuid.  We use the alternate_accessions property to redirect to the correct object.
```

Comment:
In the last sentence, is there a word which could be used instead of "release" (in "to release all identifying properties")? As a wrangler I do understand what that means, but I wonder if it is a bit confusing since "release" also refers to a `status` of objects.